### DISCUSSION TITLE
Is there a way to render OpenAPI Specs (OAS) file?

### DISCUSSION THREAD

**Discussion by eo33** [squidfunk/mkdocs-material#7776] [Q&A] [answered]
Hi everyone 👋 I'm trying to render my OpenAPI Spec (OAS) here in MKDocs-material. I've done some research and I found a plugin called `neoteroi-mkdocs` that I can use to render my OAS file. This is what it looks like:

![image](https://github.com/user-attachments/assets/a763fb98-2b50-48d7-81b2-a21834d7a344)

I wonder if there's a built-in library from MKDocs material that allows me to render the OAS natively.

**Comment by kamilkrzyskow** [comment, marked as answer]
MkDocs relies on the plugin environment and installing 3rd party plugins is the way to go in this case. IMO the Material theme doesn't have to maintain every possible scenario and become a hub for the most useful tools.

This is basically a duplicate of:
- https://github.com/squidfunk/mkdocs-material/discussions/7778

There is no need for spamming, so I will lock this discussion. If you want to comment on why the currently available plugins don't meet your requirements, please do so in that other discussion, but any issues should be taken upstream nonetheless.
